cql3/Cql.g

```diff
@@ -1232,6 +1233,26 @@ dropRoleStatement returns [std::unique_ptr<drop_role_statement> stmt]
       { $stmt = std::make_unique<drop_role_statement>(name, if_exists); }
     ;
 
+named_parameter[std::unordered_map<sstring, sstring> &params]
+    : k1=cident '=>' v1=constant
+    {
+        params.emplace(k1->to_string(), v1->get_raw_text());
+    }
+    ;
+
+/**
+  * CALL proc(name1 => value, name2 => value2, ...);
```

Comment:
Is there prior art for this syntax? Please add to changelog.

Reply:
Also, document in cql-extensions.md.

Reply:
I did not see an answer.

Reply:
ah, I added an entry to extensions and marked resolved, sorry
Where do I find the changelog? Which one?